pyriemann/utils/distance.py

```diff
@@ -58,9 +82,11 @@ def distance_euclid(A, B):
     Returns
     -------
     d : float
-        Euclidean distance between A and B.
+        Log-det distance between A and B.
     """
-    return np.linalg.norm(A - B, ord='fro')
+    return np.sqrt(np.log(np.linalg.det(
+        (A + B) / 2.0)) - 0.5 *
+        np.log(np.linalg.det(A)*np.linalg.det(B)))
```

Comment:
This should use slogdet function 

https://numpy.org/doc/stable/reference/generated/numpy.linalg.slogdet.html

which is more stable